package.json

```diff
@@ -2029,6 +2029,12 @@
                     "default": true,
                     "description": "%mssql.connectionManagement.rememberPasswordsUntilRestart%"
                 },
+                "mssql.query.summaryFetchThreshold": {
```

Comment:
Is there a similar setting in ADS for this? Would it make sense to re-use the in-memory sorting/filtering limit?

Reply:
Updated to use that setting variable.